electron.Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM hokusai_force as data
```

Comment:
Is `<...>.Dockerfile` the accepted naming convention? I feel like I've seen other styles like `Dockerfile.dev` but am not sure what's preferable.

Reply:
Honestly, I prefer `Dockerfile.dev`. I am at odds with syntax highlighting, it only works when `Dockerfile` is the suffix.